### DISCUSSION TITLE
Adding "Aptos" as a viable alternative font

### DISCUSSION THREAD

**Discussion by jondotgov** [gc-da11yn/gc-da11yn.github.io#636] [Content (Open Discussion)]
A lot of agencies and departments have chosen to move to Aptos as the default font for all copy and text. As long as you can confirm that it meets or exceeds your requirements to join such giants as "Arial, Verdana, and Calibri", I'd recommend adding Aptos to the list, perhaps even mentioning that many departments/agencies are already switching to it, and it's supported by AAACT

**Comment by shawnthompson** [comment]
Thanks for opening this discussion @jondotgov. We have already started looking into adding it to our documents.

You can see the progress in issue #631.

I will leave this discussion open just in case people want to talk about the decision to move to Aptos.
